### PR TITLE
feat(alerts): Display disabled or muted state on alert list

### LINKED ISSUE #55099: [FE] Show the disabled and muted state in the issue list view
In the "status" column show whether an alert rule is disabled or muted.

Image

### PR DESCRIPTION
**Pull request by scttcper**
fixes #55099

- Displays if an issue alert is disabled or muted
- If the alert is disabled and muted it shows disabled

Screenshot 2023-08-30 at 2 49 58 PM
Screenshot 2023-08-30 at 3 11 43 PM

### CODE REVIEW

File: static/app/types/alerts.tsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ export interface IssueAlertRule extends UnsavedIssueAlertRule {
   id: string;
   projects: string[];
   snooze: boolean;
+  status: 0 | 1;
```
Comment (malwilley) [inline static/app/types/alerts.tsx:85]: Can we add a comment or make this an enum to describe what the numbers mean?

File: fixtures/js-stubs/projectAlertRule.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export function ProjectAlertRule(params: Partial<IssueAlertRule> = {}): IssueAle
     projects: [],
     snooze: false,
     frequency: 1,
+    status: 0,
```
Comment (ceorourke) [inline fixtures/js-stubs/projectAlertRule.ts:26]: Why is this only 0? Shouldn't it be 0 or 1?

Comment (scttcper) [reply fixtures/js-stubs/projectAlertRule.ts:26]: this is the stub we use in tests, setting it to enabled

Comment (ceorourke) [reply fixtures/js-stubs/projectAlertRule.ts:26]: oooh ok

**Review by ceorourke — APPROVED** [review]
looks good to my uneducated backend eyes as long as you merge after my pr

**Pull request merged**